### PR TITLE
Initialized db-migrate

### PR DESCRIPTION
**Pull request by seb879**
Create files and configure db-migrate

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "dev": {
+      "host": "localhost",
+      "user": { "ENV" : "DB_USER" },
+      "password" : { "ENV" : "DB_PASS" },
+      "database": "itsMyCity",
+      "driver": "mysql",
+      "multipleStatements": true
+    }
+  }
```
Comment (erabin05) [inline database.json]: space !!!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# next.js build output
+.next
```
Comment (erabin05) [inline .gitignore]: space !!!

**Review by erabin05 — COMMENTED** [review]
- Need space

File: database.json (hunk 1 above)
Comment (seb879) [reply database.json]: fixed

File: .gitignore (hunk 2 above)
Comment (seb879) [reply .gitignore]: fixed

**Comment by seb879** [comment]
add and configure Eslint and add a line to the end of files

**Review by erabin05 — APPROVED** [review]
Approved

**Pull request merged**